builder/builder-next/worker/worker_test.go

```diff
@@ -0,0 +1,79 @@
+package worker
+
+import (
+	"testing"
+
+	"github.com/containerd/platforms"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWorker_Platforms_SetPlatforms(t *testing.T) {
```

Comment:
Quite some boiler plating in these tests - wondering what it looks like with a table-test (but haven't tried)

Reply:
Good idea, done; using table-style testing makes it easier to understand.